AspNetCore.Diagnostics.HealthChecks.sln

```diff
@@ -195,7 +195,6 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".Solution Items", ".Solution Items", "{6D686167-7E39-44C2-B914-CC09B5CF226A}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
-		.gitignore = .gitignore
```

Comment:
?

Reply:
No clue how I managed to remove that, woops 😅